             vcfinfo    25   2018-04-24 15:37:19,088   Initialized
             vcfinfo    75   2018-04-24 15:37:19,088   Running
             vcfinfo    35   2018-04-24 15:37:19,090   no such table: variant_annotator
Traceback (most recent call last):
  File "C:\Users\Kyle\git\CRAVAT-newarch\pipdir\cravat\base_postaggregator.py", line 77, in run
    self.base_setup()
  File "C:\Users\Kyle\git\CRAVAT-newarch\pipdir\cravat\base_postaggregator.py", line 128, in base_setup
    self._alter_tables()
  File "C:\Users\Kyle\git\CRAVAT-newarch\pipdir\cravat\base_postaggregator.py", line 152, in _alter_tables
    self.cursor_w.execute(q)
sqlite3.OperationalError: no such table: variant_annotator
